app/models/mortgage_calculator/stamp_duty.rb

```diff
@@ -79,5 +70,38 @@ def percentage_tax
       result = (tax_due / price) * 100
       result.nan? ? 0 : result
     end
+
+    def second_home?
+      buyer_type == 'isSecondHome'
+    end
+
+    private
+
+    def add_tax_band(band, rate)
```

Comment:
can we have a different name for this method, perhaps `tax_for_band`, `band_tax`